### PR TITLE
AocodaRCF7Mini Target

### PR DESCRIPTION
**Pull request by Ultraexpert**
please help this target make the next build

### CODE REVIEW

**Review by MrD-RC — CHANGES_REQUESTED** [review]
You have made changes to the SPEEDYBEEF7V2 and SPEEDYBEEF7V3 targets. Those changes should be reverted.

File: src/main/target/SPEEDYBEEF7V2/target.c [hunk 1]
```diff
@@ -1,42 +0,0 @@
...
-    DEF_TIM(TIM1, CH2, PA9,     TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO, 0, 1),  // S2
-    DEF_TIM(TIM1, CH3, PA10,    TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO, 0, 1),  // S3
-    DEF_TIM(TIM3, CH4, PC9,     TIM_USE_MC_MOTOR | TIM_USE_FW_MOTOR, 0, 0),  // S4
-
-    DEF_TIM(TIM8, CH2N, PB0,  TIM_USE_LED, 0, 0),    // LED
-
-    DEF_TIM(TIM5, CH1, PA0,  TIM_USE_ANY, 0, 0), // Camera Control
-};
-
-const int timerHardwareCount = sizeof(timerHardware) / sizeof(timerHardware[0]);
```
Comment (MrD-RC) [inline src/main/target/SPEEDYBEEF7V2/target.c]: This file needs to be reinstated

File: src/main/target/SPEEDYBEEF7V3/CMakeLists.txt [hunk 2]
```diff
@@ -1 +1 @@
-target_stm32f722xe(SPEEDYBEEF7V3)
\ No newline at end of file
+<
```
Comment (MrD-RC) [inline src/main/target/SPEEDYBEEF7V3/CMakeLists.txt]: This change needs to be undone.

File: src/main/target/SPEEDYBEEF7V3/config.c [hunk 3]
```diff
@@ -25,10 +25,8 @@
 #include "platform.h"
 
 #include "fc/fc_msp_box.h"
+
 #include "io/serial.h"
 
 void targetConfiguration(void)
 {
-    serialConfigMutable()->portConfigs[findSerialPortIndexByIdentifier(SERIAL_PORT_USART4)].functionMask = FUNCTION_ESCSERIAL;
-    pinioBoxConfigMutable()->permanentId[0] = BOX_PERMANENT_ID_USER1;
```
Comment (MrD-RC) [inline src/main/target/SPEEDYBEEF7V3/config.c]: Again, changes here need to be undone.

File: src/main/target/SPEEDYBEEF7V3/target.c [hunk 4]
```diff
@@ -20,30 +20,4 @@
...
-    DEF_TIM(TIM4, CH2, PB7,     TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO, 0, 0),  // S5
-    DEF_TIM(TIM3, CH2, PB5,     TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO, 0, 0),  // S6 Clash with S2, DSHOT does not work
-    DEF_TIM(TIM3, CH3, PB0,     TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO, 0, 0),  // S7
-    DEF_TIM(TIM3, CH4, PB1,     TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO, 0, 0),  // S8
-
-    DEF_TIM(TIM8, CH3, PC8,  TIM_USE_LED, 0, 0),    // LED
-    DEF_TIM(TIM5, CH1, PA0,  TIM_USE_ANY, 0, 0), // Camera Control
-};
-
-const int timerHardwareCount = sizeof(timerHardware) / sizeof(timerHardware[0]);
```
Comment (MrD-RC) [inline src/main/target/SPEEDYBEEF7V3/target.c]: This file needs to be put back to how it was before your PR too.

File: src/main/target/SPEEDYBEEF7V3/target.h [hunk 5]
```diff
@@ -23,152 +23,45 @@
...
 #define USE_ESC_SENSOR
 #define USE_SERIAL_4WAY_BLHELI_INTERFACE
 
-#define SERIALRX_UART           SERIAL_PORT_USART2
-#define DEFAULT_RX_TYPE         RX_TYPE_SERIAL
-#define SERIALRX_PROVIDER       SERIALRX_SBUS
-
-#define MAX_PWM_OUTPUT_PORTS        8
-
-#define CURRENT_METER_SCALE         490
```
Comment (MrD-RC) [inline src/main/target/SPEEDYBEEF7V3/target.h]: Also, this file needs to go back to how it was.

**Review by MrD-RC — CHANGES_REQUESTED** [review]
Files for the SPEEDYBEEF7V2 and SPEEDYBEEF7V2 targets have been modified. These changes need to be reverted back to how they were before. It looks like the code has been cut and pasted to create your new target. You shouldn't change existing targets when submitting a new one.

File: src/main/target/SPEEDYBEEF7V3/target.h (hunk 5 above)
Comment (Ultraexpert) [reply src/main/target/SPEEDYBEEF7V3/target.h]: Thank you for your help.

**Pull request closed without merging**